packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -225,242 +235,273 @@ contract Initializer_Test is Bridge_Initializer {
             })
         );
         // DataAvailabilityChallengeImpl
         contracts.push(
             InitializeableContract({
+                name: "DataAvailabilityChallenge",
                 target: deploy.mustGetAddress("DataAvailabilityChallenge"),
-                initCalldata: abi.encodeCall(dataAvailabilityChallenge.initialize, (address(0), 0, 0, 0, 0)),
-                initializedSlotVal: deploy.loadInitializedSlot("DataAvailabilityChallenge")
+                initCalldata: abi.encodeCall(dataAvailabilityChallenge.initialize, (address(0), 0, 0, 0, 0))
             })
         );
         // DataAvailabilityChallengeProxy
         contracts.push(
             InitializeableContract({
+                name: "DataAvailabilityChallengeProxy",
                 target: address(dataAvailabilityChallenge),
-                initCalldata: abi.encodeCall(dataAvailabilityChallenge.initialize, (address(0), 0, 0, 0, 0)),
-                initializedSlotVal: deploy.loadInitializedSlot("DataAvailabilityChallengeProxy")
+                initCalldata: abi.encodeCall(dataAvailabilityChallenge.initialize, (address(0), 0, 0, 0, 0))
+            })
+        );
+        // AnchorStateRegistry
+        contracts.push(
+            InitializeableContract({
+                name: "AnchorStateRegistry",
+                target: address(anchorStateRegistry),
+                initCalldata: abi.encodeCall(
+                    anchorStateRegistry.initialize,
+                    (new AnchorStateRegistry.StartingAnchorRoot[](1), SuperchainConfig(address(0)))
+                )
             })
         );
+        // AnchorStateRegistryProxy
+        contracts.push(
+            InitializeableContract({
+                name: "AnchorStateRegistryProxy",
+                target: address(anchorStateRegistry),
+                initCalldata: abi.encodeCall(
+                    anchorStateRegistry.initialize,
+                    (new AnchorStateRegistry.StartingAnchorRoot[](1), SuperchainConfig(address(0)))
+                )
+            })
+        );
+
+        // Nicknamed contracts.
+        nicknames["OptimismPortal2Proxy"] = "OptimismPortalProxy";
     }
 
     /// @notice Tests that:
-    ///         1. All `Initializable` contracts in `src/L1` and `src/L2` are accounted for in the `contracts` array.
-    ///         2. The `_initialized` flag of each contract is properly set to `1`, signifying that the
-    ///            contracts are initialized.
-    ///         3. The `initialize()` function of each contract cannot be called more than once.
+    ///         1. All `Initializable` contracts in `src/` (except periphery) are accounted for in `contracts`.
+    ///         2. The `_initialized` flag of each contract is properly set.
+    ///         3. The `initialize()` function of each contract cannot be called again.
     function test_cannotReinitialize_succeeds() public {
```

Comment:
Function is completely updated, recommend reading the actual function instead of the diff.